### ISSUE TITLE
v0.0.24 - Full screen displays but "Checking" progress bar never displays

### ISSUE THREAD

**Issue by LGBudd** [Portree-Kid/flightgear-airports#89]
The entire screen displays correctly now (problem with asphalt layer removed) but a click on "Check" never shows a progress bar and doesn't complete. Could there be something wrong with my install?
![2020-08-08](https://user-images.githubusercontent.com/19491531/89722756-fa010b80-d9b2-11ea-9e90-ec831cb5ca41.png)
The debugger shows the following:
`Failed to load resource: net::ERR_CONNECTION_REFUSED  http://localhost:8080/json/ai/models?d=99`
Then many "NewClass" renderer.js:11 lines
Then:
```
TypeError: Cannot read property 'eachLayer' of undefined
    at a.check (renderer.js:11)
    at a.showCheck (renderer.js:11)
    at W (renderer.js:1)
    at a.o (renderer.js:1)
    at W (renderer.js:1)
    at a.e.$emit (renderer.js:1)
    at a.click (renderer.js:11)
    at HTMLAnchorElement.<anonymous> (renderer.js:1)
```
﻿
​

**Comment by wkitty42** [comment]
that ERR_CONNECTION_REFUSED is because you don't have FG running so FGA can query it for where the AI craft are located... no idea about the rest, though...

**Comment by Portree-Kid** [comment]
The reason is, that obviously my check for a displayed APT dat overlay isn't working. It's required for the on runway check.

**Comment by LGBudd** [comment]
Thank you both! I was able to at least work around as @wkitty42 stated above. FlightGear had to be running the same airport I was trying to check, but that made the check work. I had also rescanned as I updated to 2020.2.1 and thought that might make a difference, since it did not work when I got FlightGear running but at an airport different than the one I wanted had loaded in FGA. I'm glad to hear that it is the way Keith indicates, however, as it makes it much "cleaner."
